core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/SnowflakeExpressionBuilder.scala

```diff
@@ -753,17 +753,71 @@ class SnowflakeExpressionBuilder(override val vc: SnowflakeVisitorCoordinator)
       Option(ctx.NOT()).fold[ir.Expression](between)(_ => ir.Not(between))
   }
 
-  override def visitPredIn(ctx: PredInContext): ir.Expression = errorCheck(ctx) match {
-    case Some(errorResult) => errorResult
-    case None =>
-      val in = if (ctx.subquery() != null) {
-        // In the result of a sub query
-        ir.In(ctx.expression().accept(this), Seq(ir.ScalarSubquery(ctx.subquery().accept(vc.relationBuilder))))
-      } else {
-        // In a list of expressions
-        ir.In(ctx.expression().accept(this), ctx.exprList().expr().asScala.map(_.accept(this)))
```

Comment:
You have deleted the error checking match. That needs to go back in so that errorNodes are generated

Reply:
done